lib/vquic/quiche.c

```diff
@@ -100,6 +100,7 @@ static CURLcode qs_disconnect(struct quicsocket *qs)
     qs->cfg = NULL;
   }
   if(qs->conn) {
+    (void)quiche_conn_close(qs->conn, TRUE, 0, NULL, 0);
```

Comment:
I tested but the result is same. This only set the connection state to be closed, you need to run a usual event processing - see https://docs.rs/quiche/0.6.0/quiche/struct.Connection.html#method.close

Reply:
ah right, I forgot about that...